<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mkoq_rjsxx</class>
 <widget class="QDialog" name="mkoq_rjsxx">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mkoq_rjsxx</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="mkoq_rjsxx_xxtitle_l">
     <property name="text">
      <string>mkoq_registracionni jurnal (RJ)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="mkoq_rjsxx_xxprg_l">
     <property name="text">
      <string>=mkoq_rjsxx=</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="mkoq_rjsxx_exit_c">
     <property name="text">
      <string>vixod</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="mkoq_rjsxx_b1">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <property name="verticalSpacing">
       <number>20</number>
      </property>
      <item row="5" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="mkoq_rjsxx_kr_l">
          <property name="text">
           <string>mko_Kr</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mkoq_rjsxx_kr_e">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mkoq_rjsxx_krn_e">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="mkoq_rjsxx_dokk_l_2">
        <property name="text">
         <string>mko_data operacii</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="mkoq_rjsxx_rub_l">
        <property name="text">
         <string>mko_summa</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="mkoq_rjsxx_dokk_l">
        <property name="text">
         <string>mko_dokument</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QLineEdit" name="mkoq_rjsxx_dokk_e">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mkoq_rjsxx_dokn_e">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="mkoq_rjsxx_dokd_l">
          <property name="text">
           <string>ot</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mkoq_rjsxx_dokd_e">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="mkoq_rjsxx_rub_l_2">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="mkoq_rjsxx_to_e">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="mkoq_rjsxx_db_l">
        <property name="text">
         <string>mko_Db</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLineEdit" name="mkoq_rjsxx_data_e">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QLineEdit" name="mkoq_rjsxx_db_e">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mkoq_rjsxx_dbn_e">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="mkoq_rjsxx_rub_e">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="mkoq_rjsxx_to_l">
        <property name="text">
         <string>mko_operacia</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="mkoq_rjsxx_grid_c">
       <property name="text">
        <string>vibrat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mkoq_rjsxx_back_c">
       <property name="text">
        <string>nazad</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mkoq_rjsxx_next_c">
       <property name="text">
        <string>vpered</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="mkoq_rjsxx_xxsma_l">
       <property name="text">
        <string>=Мялик К.О.=</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mkoq_rjsxx_data_e</tabstop>
  <tabstop>mkoq_rjsxx_dokk_e</tabstop>
  <tabstop>mkoq_rjsxx_dokn_e</tabstop>
  <tabstop>mkoq_rjsxx_dokd_e</tabstop>
  <tabstop>mkoq_rjsxx_to_e</tabstop>
  <tabstop>mkoq_rjsxx_db_e</tabstop>
  <tabstop>mkoq_rjsxx_dbn_e</tabstop>
  <tabstop>mkoq_rjsxx_kr_e</tabstop>
  <tabstop>mkoq_rjsxx_krn_e</tabstop>
  <tabstop>mkoq_rjsxx_rub_e</tabstop>
  <tabstop>mkoq_rjsxx_grid_c</tabstop>
  <tabstop>mkoq_rjsxx_back_c</tabstop>
  <tabstop>mkoq_rjsxx_next_c</tabstop>
  <tabstop>mkoq_rjsxx_exit_c</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
